blackjax/mcmc/rmh.py

```diff
@@ -74,35 +73,6 @@ def init(position: PyTree, logdensity_fn: Callable) -> RMHState:
     return RMHState(position, logdensity_fn(position))
 
 
-def kernel():
```

Comment:
This `kernel` function should mimic the `hmc_proposal` function in `hmc.py` (more info below).